lib/src/base-styles/_nx-tables.scss

```diff
@@ -183,6 +183,13 @@
   -ms-user-select: none;
 }
 
+.nx-cell--header--filter {
```

Comment:
I don't think you can chain BEM modifiers like this, I think what you'd want is something like `.nx-cell--filter-header`.  Or maybe it should even be a modifier on the row instead of the invididual cells: `.nx-table-row--filter-header`

Reply:
Done: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/4f2e827522b2c9ccfec400e47e4f3529cb43259d

Reply:
In that commit you've got it as `nx-table-cell--filter-header`.  Our relevant base class there is `nx-cell`, not `nx-table-cell`.

Reply:
Fixed: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/833227234bf35946b70c3bb93cf9d63eaf6c97f8